packages/jstz-types/index.d.ts

```diff
@@ -103,17 +103,23 @@ declare interface Response extends Body {
 
 declare var Response: {
   readonly prototype: Response;
-  new (body?: BodyInit | null, init?: ResponseInit): Response;
+  new(body?: BodyInit | null, init?: ResponseInit): Response;
   json(data: unknown): Response;
   error(): Response;
 };
 
-declare interface TextEncoder {
-  atob(data: string): string;
-  btoa(data: string): string;
```

Comment:
We still want to retain `atob` and `btoa` -- please add these to the global scope.

Reply:
can they be expressed in term of TextEncoder.encode ?

Reply:
Maybe, but we already have existing implementations in Rust that are compliant with their Node & Deno equivalents